singlem/chainsaw.py

```diff
@@ -25,9 +25,9 @@ def chainsaw(**kwargs):
         if not input_spkg.is_protein_package():
             raise Exception("Only works with protein packages.")
 
-        # Ensure v2
-        if input_spkg.version != 2:
-            raise Exception("Only works with v2 packages for the moment.")
+        # Ensure v2+
+        if input_spkg.version < 2:
+            raise Exception("Only works with v2+ packages for the moment.")
```

Comment:
I reckon it'd be better to write `if input_spkg.version in [2,3]` because then if v4 comes along we don't automatically pretend to support it when actually it isn't supported.